gcloud-java-contrib/gcloud-java-nio/src/test/java/com/google/gcloud/storage/contrib/nio/CloudStorageFileSystemProviderTest.java

```diff
@@ -547,115 +633,120 @@ public void testIsDirectory() throws IOException {
   }
 
   @Test
-  public void testIsDirectory_trailingSlash_alwaysTrue() {
-    assertThat(Files.isDirectory(Paths.get(URI.create("gs://military/fundir/")))).isTrue();
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Method names should not contain underscores](https://www.codacy.com/app/mziccard/gcloud-java/file/2333923614/issues/source?bid=3123107&fileBranchId=3190440#l636)